test/unit/annotation_storage_spec.js

```diff
@@ -50,12 +54,10 @@ describe("AnnotationStorage", function () {
         called = true;
       };
       annotationStorage.onSetModified = callback;
-      annotationStorage.getOrCreateValue("asdf", { value: "original" });
-      expect(called).toBe(false);
 
       // not changing value
       annotationStorage.setValue("asdf", { value: "original" });
-      expect(called).toBe(false);
+      expect(called).toBe(true);
```

Comment:
We're not testing the `false` case anymore now, and the comment above this test don't match with the test anymore.